cis/Issues/digitalInsurancePassport/description.xml

```diff
@@ -0,0 +1,130 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Digital Insurance Passport</en>
+		<nl>Demo Digitaal Paspoort Verzekeren</nl>
+	</Name>
+	<ShortName>
+		<en>Demo Digital Insurance Passport</en>
+		<nl>Demo Digitaal Paspoort Verzekeren</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>cis</IssuerID>
+	<CredentialID>digitalInsurancePassport</CredentialID>
+	<Description>
+		<en>With your Digital Insurance Passport, insurers can see who you are and determine you do not appear in national and international sanctions, investigation and incident registers. This check is mandatory by law and insurers do these together to prevent costs and extra work for customers and insurers.</en>
```

Comment:
determine *if* you
but maybe use: and verify that you

do these together -> do these collectively  (although maybe just leave it at 'This check is mandatory by law')

Reply:
nice, missed that one